testsuite/functional-testsuite/src/test/java/org/sonatype/nexus/testsuite/SearchIT.groovy

```diff
@@ -154,4 +155,15 @@ extends FunctionalTestSupport
   void suite() {
     run("search/.*\\.t.js")
   }
+
+  public File resolveTestFileAnywhereInResources(final String regex) {
```

Comment:
Would be nice if we could avoid iterating over the entire set of test data files for each lookup as this will slow down CI. Is this needed because of the way the shared assets will be unpacked? Is there a way to improve the `TestDataRule` from goodies-testsupport to handle this better?